roofit/batchcompute/README.md

```diff
@@ -8,23 +8,23 @@ While fitting, a significant amount of time and processing power is spent on com
 As of ROOT v6.26, RooBatchComputes also provides multithread and [CUDA](https://en.wikipedia.org/wiki/CUDA) instances of the computation functions, resulting in even greater improvements for fitting times.
 
 ### How to use
-This library is an internal component of RooFit, as a results users are not supposed to actively interact with it. Instead, they can benefit from significantly faster times for fitting by calling `fitTo()` and providing a `BatchMode(RooBatchCompute::Cpu)` or a `BatchMode(RooBatchCompute::Cuda)` option. 
+This library is an internal component of RooFit, as a results users are not supposed to actively interact with it. Instead, they can benefit from significantly faster times for fitting by calling `fitTo()` and providing a `BatchMode("cpu")` or a `BatchMode("cuda")` option. 
 ```c++
-  // fit using the most efficient library that the computer's cpu can support
+  // fit using the most efficient library that the computer's CPU can support
   RooMyPDF.fitTo(data, BatchMode("cpu")); 
   
-  // fit using the cuda library along with the most efficient library that the computer's cpu can support
+  // fit using the CUDA library along with the most efficient library that the computer's CPU can support
   RooMyPDF.fitTo(data, BatchMode("cuda")); 
 ```
-**Note: In case the system does not support vector instructions, the `RooBatchCompute::Cpu` option is guaranteed to work properly by using a generic cpu library. In contrast, users must first make sure that their system supports cuda in order to use the `RooBatchCompute::Cuda` option. If this is not the case, an exemption will be thrown.**
+**Note: In case the system does not support vector instructions, the `RooBatchCompute::Cpu` option is guaranteed to work properly by using a generic CPU library. In contrast, users must first make sure that their system supports CUDA in order to use the `RooBatchCompute::Cuda` option. If this is not the case, an exemption will be thrown.**
```

Comment:
```suggestion
**Note: In case the system does not support vector instructions, the `RooBatchCompute::Cpu` option is guaranteed to work properly by using a generic CPU library. In contrast, users must first make sure that their system supports CUDA in order to use the `RooBatchCompute::Cuda` option. If this is not the case, an exception will be thrown.**
```